src/components/CommentsArea/Comments/index.jsx

```diff
@@ -0,0 +1,76 @@
+import doctorAvatar from '../../../assets/doctorAvatar.png';
+import userAvatar from '../../../assets/userAvatar.png';
+import SComments from './style';
+
+function Comments() {
+  return (
+    <SComments>
+      <>
+        <div className="comment userComment">
+          <div className="avatarContainer">
+            <img className="avatars" src={userAvatar} alt="User Avatar" />
+          </div>
```

Comment:
Why call a SComments being a div, to call a react fragment, to call another div again?